Classes/URKArchive.h

```diff
@@ -234,6 +234,22 @@ extern NSString *URKErrorDomain;
 + (BOOL)urlIsARAR:(NSURL *)fileURL;
 
 /**
+ *  Determines whether the current file is a RAR volume part
```

Comment:
Now, I'm thinking, since the name implied multiple, I've made `hasMultipleVolumes` to check if it actually has ***multiple*** volume (checking `listVolumeURLs`)... Now I see you mean if it is a part of a volume, even if it is left alone and no other parts of the volume are to be found (extraction will fail, obviously), so I'll just check `isVolume`.